src/sourcemap.litcoffee

```diff
@@ -179,9 +179,23 @@ Regular Base64 Encoding
         BASE64_CHARS[value] or throw new Error "Cannot Base64 encode value: #{value}"
 
 
-Our API for source maps is just the `SourceMap` class.
+Save source maps cache `filename`: `map`.
 
-    module.exports = SourceMap
+    sourceMaps = Object.create {}
 
+    registerCompiled = (filename, source, sourcemap) ->
+      if sourcemap?
+        sourceMaps[filename] = sourcemap
 
+    getSourceMap = (filename) ->
+      sourceMaps[filename]
```

Comment:
Why not just make these as additional methods on the class? And you can create the cache in the constructor, something like `@cache = Object.create {}`, if this class is a singleton; or define it at the top of the file if there are multiple instances of `SourceMap` instantiated but they should all share the same cache.